src/store/rootReducer.ts

```diff
@@ -7,15 +7,23 @@ import { environmentVariablesSlice } from "../features/environmentVariables";
 import { tabsSlice } from "../features/tabs";
 import { enviromentsSlice } from "../features/metadata";
 import { environmentCreateSlice } from "../features/environmentCreate/environmentCreateSlice";
+// TODO rename */reducer.ts files to match */*slice.ts pattern
+import { namespacesSlice } from "../features/namespaces/reducer";
+import { environmentsSlice } from "../features/environments/reducer";
+import { notificationSlice } from "../features/notification/notificationSlice";
 
 export const rootReducer = {
   [apiSlice.reducerPath]: apiSlice.reducer,
   channels: channelsSlice.reducer,
   requestedPackages: requestedPackagesSlice.reducer,
   environmentVariables: environmentVariablesSlice.reducer,
   tabs: tabsSlice.reducer,
+  // TODO: rename (fix spelling) and consolidate with environments slice
   enviroments: enviromentsSlice.reducer,
   environmentDetails: environmentDetailsSlice.reducer,
   dependencies: dependenciesSlice.reducer,
-  environmentCreate: environmentCreateSlice.reducer
+  environmentCreate: environmentCreateSlice.reducer,
+  namespaces: namespacesSlice.reducer,
+  environments: environmentsSlice.reducer,
+  notification: notificationSlice.reducer
```

Comment:
We really shouldn't have so many state slices. Not sure why it was architected this way. But this is something to address in future PRs.